content/docs/start/experiments.md

```diff
@@ -222,6 +222,19 @@ $ dvc exp show --drop 'Created|train|loss'
  ─────────────────────────────────────────────────────
 ```
 
+<details>
+
+### ℹ️ More information about metrics
+
+Metrics are what you use to evaluate your models. DVC allows any scalar values
+to be used as metrics. It's able to track the metrics we defined in the code
+with the Keras integration introduced recently. You can also specify a custom
+metrics file using `--metrics` option of `dvc exp init`, and write the metrics
```

Comment:
also edit `dvc.yaml`. 

I think if we keep this section we should say something like - To capture project metrics in DVC you use JSON, TSV, whatever file or multiple files, specify them as metrics files in dvc.yaml , and your app writes with arbitrary structure into them. There are a bunch of commands (like `dvc exp show`) or services (mentioned Studio?) that interpret those files ...



Essense of the metrics, less abstract repetitive things

Reply:
Also the Keras integration is completely out of context here. Either link to the DVCLive page (I assume that's what it refers to?) or avoid! 🙂